app/Http/Controllers/UserController.php

```diff
@@ -2,10 +2,94 @@
 
 namespace App\Http\Controllers;
 
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Validator;
+
 class UserController extends Controller
 {
     public function index()
     {
-        return view('auth.user');
+        $user = Auth::user();
+
+        return view('auth.user', ['user' => $user])
+            ->with('neptun', $user->educationalInformation->neptun)
```

Comment:
Not all users have educationalInformation, for them it fails. Also, you do not have to pass any of these values, as you have access to Auth::user() in the blade.